src/seedu/addressbook/ui/TextUi.java

```diff
@@ -18,16 +18,10 @@
  */
 public class TextUi {
 
-    /** A decorative prefix added to the beginning of lines printed by AddressBook */
-    private static final String LINE_PREFIX = "|| ";
-
-    /** A platform independent line separator. */
-    private static final String LS = System.lineSeparator();
-
     private static final String DIVIDER = "===================================================";
```

Comment:
This could be moved to the Formatter as well? May be even things like this should be moved to the Formatter? Note that it may not be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.